src/binary/ibc-setup/commands/keys-generate.ts

```diff
@@ -0,0 +1,32 @@
+import fs from 'fs';
+
+import { generateMnemonic } from '../utils/generate-mnemonic';
+import { resolveOption } from '../utils/resolve-option';
+
+export type Options = {
+  keyFile?: string;
+};
+
+export function keysGenerate(flags: Options) {
+  const options = {
+    keyFile: resolveOption(
+      'key-file',
```

Comment:
`resolveOption` no longer takes an identifier as the first arg (properly), so this should be removed (now it means the option value will always be `key-file`, correct?).

Maybe to avoid such a mistake in the future, you could do something like `requireOption('key-file')(flags.keyFile, process.env.RELAYER_KEY_FILE)` easier to remove the extra call than a compatible arg when doing a refactor

Reply:
Indeed, this is a bug. `resolveOption` doesn't take the identifier anymore but I curried the `resolveRequiredOption`: 45a6278